### PR TITLE
Add support to jk01 keyboard 

### PR DESCRIPTION
**Pull request by gskygithub**
## Description

Add jukaie/jk01 keyboard

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/20573

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [ ] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/jukaie/jk01/jk01.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  "matrix": {"rows": 14, "cols": 8},
+  "customKeycodes": [
+    {"name": "Mission Control", "title": "Mission Control in macOS", "shortName": "Mission Control"},
+    {"name": "Launch Pad", "title": "Launch Pad in macOS", "shortName": "Launch Pad"},
+    {"name": "Left Option", "title": "Left Option in macOS", "shortName": "Left Option"},
+    {"name": "Right Option", "title": "Right Option in macOS", "shortName": "Right Option"},
+    {"name": "Left Cmd", "title": "Left Command in macOS", "shortName": "Left Command"},
+    {"name": "Right Cmd", "title": "Right Command in macOS", "shortName": "Right Command"},
+    {"name": "Task View", "title": "Task View in windows", "shortName": "Task View"},
+    {"name": "File Explorer", "title": "File Explorer in windows", "shortName": "File Explorer"}
```
Comment (Xelus22) [inline v3/jukaie/jk01/jk01.json:16]: I don't see any custom keycodes in your QMK source files.
Please remove these.

**Review by Xelus22 — CHANGES_REQUESTED** [review]

**Review by Xelus22 — CHANGES_REQUESTED** [review]
image

please colour in the KLE https://www.caniusevia.com/docs/layouts#kle-json-rules

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Still havent fixed it
image

**Comment by gskygithub** [comment]
> Still havent fixed it image

@Xelus22 For me, it is looking normal in VIA.
image

**Comment by Xelus22** [comment]
Please follow the VIA docs. Alphas are cccccc, and the mods and accents. It looks different when you go to settings and change the theme.

**Comment by gskygithub** [comment]
@Xelus22 Resolved requested changes, please take a look, thanks.

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**